app/api/api_v1/endpoints/species.py

```diff
@@ -47,6 +47,47 @@ def read_species_by_search(
     return species
 
 
+@router.post("/fuzzymatch/", response_model=List[schemas.SpeciesSummary])
+def read_fuzzy_species_by_search(
+    query_str: str,
+    db: Session = Depends(deps.get_db),
+    min_string_similarity_score: float = 0.3,
+    limit: int = 0,
+    order_by: Optional[List[str]] = Query(None),
+) -> Any:
+
+    expressions_dict: dict = {
+        "join": "OR",
+        "expressions": [
+            {
+                "column_name": "matched_canonical_full_name",
+                "search_term": query_str,
+                "operator": "eq",
+                "fuzzy": True,
+                "min_string_similarity": min_string_similarity_score,
+            },
+            {
+                "column_name": "current_name",
+                "search_term": query_str,
+                "operator": "eq",
+                "fuzzy": True,
+                "min_string_similarity": min_string_similarity_score,
+            },
+        ],
+    }
+
+    expressions: schemas.ExpressionGroup = schemas.ExpressionGroup(**expressions_dict)
```

Comment:
Just a hint: explicit typing here is not necessary because it's inferred from the assignment.

Reply:
Ohh. Okay, thanks. I will update it.